src/services/services.ts

```diff
@@ -283,6 +283,12 @@ namespace ts {
         }
     }
 
+    enum ContextualDocumentationType {
+        None = "None",

```

Comment:
Nit: we prefer numeric const enums for efficiency

Reply:
Seems `Map` interface cannot work with the numerical enum, does it?

Reply:
It doesn't need to be map.. Jus two properties that store array of decl comments one for get and one for set accessor ?

Reply:
Okay...but I felt maybe it will take more contextual info in the future. For example getter or setter in LHS or RHS

Reply:
But never mind, We could refactor in time.